src/branch_view.js

```diff
@@ -46,7 +48,9 @@ function renderHtmlBlob(parent, html, hasApp) {
  * @param {Object} storage
  * @param {Boolean} hasApp
  */
-branch_view.handleBranchViewData = function(server, branchViewData, requestData, storage, hasApp, testFlag) {
+branch_view.handleBranchViewData = function(refToBranchObj, server, branchViewData, requestData, storage, hasApp, testFlag) {
+    journeys_utils.refToBranchObj = refToBranchObj;
```

Comment:
On second thought, I would just call this branch, similar to how we do in other places where it's passed in. I think it will be easier to read.